### ISSUE TITLE
(/ti:TextInventory[@tiid = $a_inv])[1] returns nothing

### ISSUE THREAD

**Issue by PonteIneptique** [PonteIneptique/CTS-API#21]
Structure of the DB
```
db
|---apps
|---greekLit
|---inventory
     |-----annotsrc.xml
|---latinLit
|---system
|---xq
     |-----CTS.xq
     |-----all xquery in fact
```

In [cts.xquery line 336](https://github.com/PonteIneptique/CTS-API/blob/93fe9304262efd6a028398484e05c78531ebe5d8/xquery/existDB/cts.xquery#L336) : 
```xquery
 let $ti := (/ti:TextInventory[@tiid = $a_inv])[1]
```
returns nothing. Removing attribute returns nothing as well. Accessing `/db/inventory/` seems to fail.

**Comment by PonteIneptique** [comment]
Partly linked to #22

**Comment by PonteIneptique** [comment]
Issue totally related to namespace and position of files. Fixed in @3fe7fe66aa49c02c066f8d4efb45a7438a61f2b8
